app/config/sources.inc.php

```diff
@@ -579,462 +49,1041 @@
 ];
 
 /*
-    Array structure for single website:
-    [
-       0 name,
-       1 path to local repo,
-       2 folder containing locale files,
-       3 array of supported locale,
-       4 array of supported file names,
-       5 reference locale,
-       6 url to public repo,
-       7 array of flags,
-       8 type of files (lang, raw)
-    ]
+    For each file it's possibile to specify the following fields:
+    * deadline: if the deadline is the same for all locales, assign the deadline
+      as a string in ISO format (dd-mm-yyyy):
+
+      'deadline' => '2016-04-29',
+
+      If deadline needs to change based on the locale, you can use a
+      multidimensional array, where each deadline is associated to a list of
+      locales. 'all' can be used to indicate that deadline applies to all
+      supported locales. Example, deadline only for two locales:
+
+      'deadline' => [
+          '2016-04-25' => ['de', 'fr'],
+      ],
+
+      Different deadlines:
+
+      'deadline' => [
+          '2016-04-25' => ['de', 'fr'],
+          '2016-01-05' => ['all'],
+      ],
+
+    * priority: priorities are numeric values assigned to files. 1 is the
+      highest priority, 3 is the lowest.
```

Comment:
No, there are 3, like tiers. 

They're then mapped to a 5 stars system in Webdashboard for consistency with Pontoon (1 -> 5 stars, 2 -> 3 stars, 3 -> 1 star). Priority vs stars would be backwards anyway.